tests/test_web_response.py

```diff
@@ -887,6 +887,14 @@ def test_ctor_charset_without_text():
     assert 'koi8-r' == resp.charset
 
 
+def test_ctor_content_type_with_extra():
```

Comment:
Out of curiosity, what does `ctor` stand for?

Reply:
constructor